Remove errors.Is usage

This is not supported by golang 1.11. It was introduced by golang 1.13.

===================================================================
--- a/internal/app/singularity/plugin_uninstall_linux.go
+++ b/internal/app/singularity/plugin_uninstall_linux.go
@@ -18,7 +18,7 @@
 // UninstallPlugin removes the named plugin from the system.
 func UninstallPlugin(name, libexecdir string) error {
 	err := plugin.Uninstall(name, libexecdir)
-	if errors.Is(err, os.ErrNotExist) {
+	if err == os.ErrNotExist {
 		return ErrPluginNotFound
 	}
 	if err != nil {
--- a/internal/pkg/build/sources/oci_unpack_linux.go
+++ b/internal/pkg/build/sources/oci_unpack_linux.go
@@ -163,7 +163,7 @@
 		return nil
 	})
 
-	if errors.Is(err, errRestrictivePerm) {
+	if err == errRestrictivePerm {
 		sylog.Warningf("Permission handling has changed in Singularity 3.5 for improved OCI compatibility")
 		sylog.Warningf("The sandbox will contain files/dirs that cannot be removed until permissions are modified")
 		sylog.Warningf("Use 'chmod -R u+rwX' to set permissions that allow removal")
